### PR TITLE
Quote Component Initial version without loaders - Fetch Data from The API

### PR DESCRIPTION
**Pull request by Abdulwhbhussain**
Project requirements
Use the following external API: [API Ninjas](https://api-ninjas.com/api/quotes)
Create a new component for displaying a quote
Fetch data inside of a new component
Show a loading state
Show an error state
Import the new component into your main component (next to the Calculator)
It should be on the same page as the Calculator
Note: Practice what you have learned until now: use useEffect and props.

### CODE REVIEW

File: src/components/Quote.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+          headers: {
+            'X-Api-Key': myApiKey,
+            'Content-Type': 'application/json',
+          },
+        });
+        const json = await res.json();
+        setQuote(json[0]);
+      } catch (error) {
+        setHasError(true);
+      }
```
Comment (VICTOR-CHUKWU) [inline src/components/Quote.jsx:50]: - [x] nice way to handle error :clap:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Abdulwhbhussain , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [x] All linters passed
- [ ] error and loading page handled correctly

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Quote.jsx (hunk 1 above)
Comment (Abdulwhbhussain) [reply src/components/Quote.jsx:50]: Thanks!!!

Comment (Abdulwhbhussain) [reply src/components/Quote.jsx:50]: Thanks!!!

**Review by KanzaTahreem — COMMENTED** [review]
# Approved 🙌

Hi @Abdulwhbhussain 

**_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Great work using the right GiFlow 
-  ✅ Your README looks good, well done!

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**